test/cpu_fusion.cpp

```diff
@@ -1670,32 +1670,32 @@ TEST(cpu_fusion, conv_batch_norm_folding)
 
 TEST(cpu_fusion, convbias_batch_norm_folding)
 {
-    Shape shape_input{1, 8, 3, 3};
-    Shape shape_weights{2, 8, 1, 1};
+    Shape shape_input{2, 8, 5, 5};
+    Shape shape_weights{2, 8, 2, 2};
     Shape shape_norm{2};
 
     auto make_function = [shape_input, shape_weights, shape_norm]() {
         auto input = std::make_shared<op::Parameter>(element::f32, shape_input);
         auto weights = std::make_shared<op::Parameter>(element::f32, shape_weights);
         auto bias = std::make_shared<op::Parameter>(element::f32, Shape{2});
-        double eps = 0.001;
+        double eps = 1.01;
         auto gamma = std::make_shared<op::Parameter>(element::f32, shape_norm);
         auto beta = std::make_shared<op::Parameter>(element::f32, shape_norm);
         auto mean = std::make_shared<op::Parameter>(element::f32, shape_norm);
         auto var = std::make_shared<op::Parameter>(element::f32, shape_norm);
         auto conv = std::make_shared<op::Convolution>(input, weights, Strides{1, 1}, Strides{1, 1});
         auto convbias =
             conv + std::make_shared<op::Broadcast>(bias, conv->get_shape(), AxisSet{0, 2, 3});
-        auto bn = std::make_shared<op::BatchNorm>(eps, gamma, beta, conv, mean, var);
-        auto f = make_shared<Function>(NodeVector{bn},
-                                       op::ParameterVector{input, weights, gamma, beta, mean, var});
+        auto bn = std::make_shared<op::BatchNorm>(eps, gamma, beta, convbias, mean, var);
+        auto f = make_shared<Function>(
+            NodeVector{bn}, op::ParameterVector{input, weights, bias, gamma, beta, mean, var});
         return f;
     };
 
     auto int_f = make_function();
     auto cpu_f = make_function();
 
-    test::Uniform<float> rng(-100.0f, 100.0f);
+    test::Uniform<float> rng(1.0f, 100.0f);
```

Comment:
How do you select the range of values? this is all positive values whereas you had + / - earlier. Do we need to test for -ve values?